pkg/liqo-controller-manager/virtualNode-controller/namespaceMap_lifecycle.go

```diff
@@ -0,0 +1,128 @@
+package virtualNode_controller
+
+import (
+	"context"
+	"fmt"
+	mapsv1alpha1 "github.com/liqotech/liqo/apis/virtualKubelet/v1alpha1"
+	constctrl "github.com/liqotech/liqo/pkg/liqo-controller-manager"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	ctrlutils "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+// remove Finalizer and Update the NamespaceMap
+func (r *VirtualNodeReconciler) deleteNamespaceMap(nm mapsv1alpha1.NamespaceMap) error {
```

Comment:
Please, use a more meaningful name, deleteNamespace is misleading since you are not deleting anything.

Reply:
yes, sure